worldengine/draw.py

```diff
@@ -1,6 +1,7 @@
 from PIL import Image
 import numpy
 import numpy.ma as ma
+import random
```

Comment:
Since now I have a bit more time to explain: random.randint() gives different results depending on whether you use Python 2 or 3, which will result in (slightly) different worlds for both versions. We switched to numpy.random due to this.